test/package-sources/rhocp-y.toml

```diff
@@ -0,0 +1,10 @@
+{{- if env.Getenv "RHOCP_MINOR_Y" "" -}}
```

Comment:
Is this saying "if RHOCP_MINOR_Y is empty" or is the second argument to Getenv the default value to use if the variable is not set?